lib/broccoli/ember-app.js

```diff
@@ -1180,29 +1180,35 @@ class EmberApp {
     @return {Array}
    */
   lintTestTrees() {
-    let lintedApp = this.addonLintTree('app', this._filterAppTree());
-    let lintedTests = this.addonLintTree('tests', this.trees.tests);
-    let lintedTemplates = this.addonLintTree('templates', this._templatesTree());
+    let testSupportTree = this._mergeTrees(this.addonTreesFor('test-support'));
 
-    lintedApp = processModulesOnly(new Funnel(lintedApp, {
-      srcDir: '/',
-      destDir: `${this.name}/tests/`,
-      annotation: 'Funnel (lint app)',
-    }), 'Babel: lintTree(app)');
+    return [
+      this.lintTestTree('app', this._filterAppTree()),
+      this.lintTestTree('tests', this.trees.tests),
+      this.lintTestTree('templates', this._templatesTree()),
+      this.lintTestTree('test-support', testSupportTree),
```

Comment:
I'd say we should likely just categorize this in with the build/rebuild work that has been underway for a while. We should continue reducing/removing LI specific customizations in this area (which is a major part of the core/extended refactoring work).

Closing...